Ft app-views

zbahati opened the pull request:
- Implement the design from the sneak peek wireframes.
- Styling is nice to have but focus on making sure everything functions first.

Commit:
Ft app-views

Munsa1 commented on app/views/posts/index.html.erb:
```diff
@@ -1 +1,47 @@
+      <p>Number of Posts: <%= @user.Posts_counter.nil? ? '0' : @user.Posts_counter %></p>
+    </div>
+  </li>
+  <% @posts.each_with_index do |post, index| %>
+    <%= link_to user_post_path(@user.id, post.id) do %>
+      <li class="user_all_posts">
+        <div class="user_bio">
+          <h3>Post#<%= index+1 %></h3>
+          <p><%= post.Text.size > 100 ? "#{post.Text[0..100]} ..." : post.Text %></p>
+          <div class="user_recent_posts">
+            <span>
+              Comments: <%= post.comments.count %>,
+            </span>
+            <span>
+              Likes: <%= post.Likes_counter.nil? ? '0' : post.Likes_counter %>
+            </span>
+          </div>
+        </div>
+
+        <div class="user_post_comments">
+          <% if post.comments.empty? %>
+            <div class="user_post_comment">
+              <p>No comments</p>
+            </div>
+          <% else %>
+            <% post.comments.each_with_index do |comment, index| %>
+              <div class="user_post_comment">
+                <h3><%= index + 1 %>. <%= comment.user.Name %></h3>
+                <p><%= comment.Text %></p>
```
- [ ] Its a requirement that here we use `five_recent_comments` created in the post model to render 5 most recent comments in the `app/views/posts/index.html.erb` page. Kindly do so.

Munsa1 commented on app/views/users/index.html.erb:
```diff
@@ -1 +1,16 @@
-<h1>LIST OF USERS </h1>
\ No newline at end of file
+
+<ul class="users_container">
+  <% @users.each do |user| %>
+    <%= link_to user_path(user.id) do %>
+      <li>
+        <div class="user_img">
+          <img src="<%= user.Photo %>" alt="Photo">
+        </div>
+        <div class="user_details">
+          <h2><%= user.Name %></h2>
+          <p>Number of Posts: <%= user.Posts_counter.nil? ? '0' : user.Posts_counter %></p>
+        </div>
+      </li>
+    <% end %>
+  <% end %>
```
- [ ] (Optional) Using partials is a better option as a practice instead of using `@objects.each`. Consider it for future projects

Munsa1 reviewed (changes requested):
Hi @zbahati, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Implemented Views As Given In The Wireframes :heavy_check_mark: 
Clean CodeBase :heavy_check_mark: 
Descriptive Pr :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Commit:
Ft app_views

Create a patial file called _user_profile.htm.erb in the shared folder
Working with diffrence instances to have one shared file
Fix lintes errors

# Finishes ..

prolajumokeoni reviewed (approved):
Hi @zbahati 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
 - Updated recent comments👍🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
